lib/src/revset.rs

```diff
@@ -253,6 +253,7 @@ pub enum RevsetExpression<St: ExpressionState> {
         candidates: Rc<Self>,
         count: usize,
     },
+    GreatestCommonAncestors(Rc<Self>),
```

Comment:
nit: I would move it closer to heads/roots because "gca" is DAG-based operation.